src/ui/settings/index.js

```diff
@@ -24,8 +24,12 @@ export default function setupSettings(kbnServer, server, config) {
   }
 
   function userSettingsNotFound(kibanaVersion) {
-    const message = 'Could not find user-provided settings for this version of Kibana (' + kibanaVersion + ')';
-    server.plugins.kibana.status.red(message);
+    let message;
+    if (server.plugins.elasticsearch.status.state === 'green') {
+      server.plugins.kibana.status.red('Could not find user-provided settings for this version of Kibana (' + kibanaVersion + ')');
```

Comment:
Can we change the message into a template string? Also the `message` variable is unused.